.github/workflows/formulaCI.yml

```diff
@@ -0,0 +1,86 @@
+name: Formula CI
+# Description: "Moving saphanabootstrap-formula from Travis to Github Actions"
```

Comment:
I would change this description by something like this: https://github.com/SUSE/habootstrap-formula/blob/master/.github/workflows/formula-ci.yml#L3

Reply:
I add the comment description like in the example